applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -283,12 +444,34 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 		     BT_AUDIO_EP_STATE_CODEC_CONFIGURED))
 #endif
 		{
-			ret = bt_audio_stream_qos(headsets[channel_index].headset_conn,
-						  unicast_group);
-			if (ret) {
-				LOG_ERR("Unable to set up QoS for headset %d: %d", channel_index,
-					ret);
+			if (headsets[channel_index].pres_dly_us != pres_delay) {
+				bool restart = any_active_headsets_started();
+
+				if (restart) {
+					audio_play_pause(false);
+				}
+
+				ret = bt_audio_stream_qos(headsets[channel_index].headset_conn,
```

Comment:
This only updates this one stream, right? What about the others? Say this stream has a higher minimum pres delay, then the others should be updated as well? Or is that done through this one call?

Reply:
No I thought this function updated all active streams.

/** @brief Configure Audio Stream QoS
 *
 *  This procedure is used by a client to configure the Quality of Service of
 *  streams in a unicast group. All streams in the group for the specified
 *  @p conn will have the Quality of Service configured.
 *  This shall only be used to configure unicast streams.
 *
 *  @param conn  Connection object
 *  @param group Unicast group object
 *
 *  @return 0 in case of success or negative value in case of error.
 */

Reply:
It will update for all the streams for that one connection. In our case we usually only have one stream with one conn, but we have two conns in that group. So this would only update for left for instance and not for right

Reply:
Thanks again for clarifying that for me